docs/operator-guides/delete-custom-connector.md

```diff
@@ -0,0 +1,50 @@
+---
+description: How to remove a custom connector in Setting Page
+---
+
+# Delete Custom Connectors in the Setting Page
```

Comment:
I think it would help to make it clear to customers that this a workaround. We should add a note that says API/UI support for this is coming soon and link to an issue.

Reply:
actually just saw michel's comment, I hadn't considered the tombstomb point. I agree with his view that we probably shouldn't recommend this without better behavior in the API

Reply:
I agree with Michel, adding this to our docs without a good process only create problems. Users will remove a connector with connections/source created...